docs/02.API-REFERENCE.md

```diff
@@ -397,37 +334,80 @@ jerry_cleanup (void);
 **See also**
 
 - [jerry_init](#jerry_init)
-- [jerry_init_with_user_context](#jerry_init_with_user_context)
 
 
-## jerry_get_user_context
+## jerry_get_context_data
 
 **Summary**
 
-Retrieve the pointer stored within the current context.
+Retrieve a pointer to the item stored within the current context by the given manager.
+
+**N.B.:** Since internally the pointer to a manager's context data item is linked to the next such pointer in a linked
```

Comment:
I noticed that the trend is to use `*Note*:` and then indent the rest, so I'll move to that.